### PR TITLE
README Instructions for Windows, Mac, and Ubuntu dev environments

### PR DESCRIPTION
**Pull request by booellean**
This is the most updated file without the extra css. The Instructions for Windows 10 vs Mac/Ubuntu have been separated per the last request: https://github.com/shescoding/shescoding-dot-org/pull/167

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,16 +331,3 @@ $ bundle exec rspec -fd
 Notes:
 + ```-fd``` stands for full details. You can also run tests by using commands ```bundle exec rspec``` or ```bundle exec rake``` to get an overview without details of each test.
 + If you are having trouble running bundle install, please make sure Nokogiri gem is installed on your machine. Full instructions can be found [here](http://www.nokogiri.org/tutorials/installing_nokogiri.html).
-
-### Running the linters
```
Comment (cherchezlafemme) [inline README.md]: we would want to keep this section. we shouldn't delete it

File: README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Yes! *She's Coding* is built by the community, for the community. We currently h
 
 ### Installation
 
-+ Install [Homebrew](http://brew.sh/) (OSX):
+#### Mac OSX: Install [Homebrew](http://brew.sh/):
```
Comment (cherchezlafemme) [inline README.md]: I would suggest make 2 lines.

File: README.md [hunk 3]
```diff
@@ -119,6 +124,162 @@ resource provided above.
...
+  Go to [Windows App Store](https://www.microsoft.com/store/productId/9PJN388HP8C9) and download the most stable version of Ubuntu, then
+
+    + Go to developer settings by searching "Developer"
+
+    + In the results under *Settings*, click  *For developer settings*
+
+    + Select *Developer Mode*
+
+  Open the Ubuntu app and complete setup (username, password, etc.), then run:
+
```
Comment (cherchezlafemme) [inline README.md]: this section doesn't seem to be formatted correctly 
styling

File: README.md [hunk 4]
```diff
@@ -119,6 +124,162 @@ resource provided above.
     $ git remote add upstream https://github.com/shescoding/shescoding-dot-org.git
     ```
 
++ To complete installation, skip past the directions for Windows 10 [here](#mac,-ubuntu,-and-windows)
```
Comment (cherchezlafemme) [inline README.md]: [Go here](#mac-ubuntu-and-windows) to complete installation, skip past the directions for Windows 10.

the link was no working

File: README.md [hunk 5]
```diff
@@ -119,6 +124,162 @@ resource provided above.
...
+---
+---
+---
```
Comment (cherchezlafemme) [inline README.md]: looks heavy

File: README.md [hunk 6]
```diff
@@ -119,6 +124,162 @@ resource provided above.
...
+    host: localhost
+    ```
+
+    + Under **development:**
+
+    ```
+    username: YourPostGresUserName
+    password: YourPostGresPassword
+    ```
++ **With PGAdmin4 still running**, complete setup by following the steps [below](#mac,-ubuntu,-and-windows).
```
Comment (cherchezlafemme) [inline README.md]: `#mac-ubuntu-and-windows` is the correct link

File: README.md [hunk 7]
```diff
@@ -14,7 +14,7 @@ Yes! *She's Coding* is built by the community, for the community. We currently h
 
 ### Installation
```
Comment (cherchezlafemme) [inline README.md]: we can update the headers to be ##, ###, #### if needed to make section separation more obvious. I am not sure the three lines are a good solution. it looks a little heavy. if we decide to stick to 3 lines - we should add them before installation too. however I am not sold on that :/

File: README.md (hunk 1 above)
Comment (booellean) [reply README.md]: This has been fixed

File: README.md (hunk 3 above)
Comment (booellean) [reply README.md]: This has been fixed

File: README.md (hunk 4 above)
Comment (booellean) [reply README.md]: This has been fixed

File: README.md (hunk 6 above)
Comment (booellean) [reply README.md]: This has been fixed (strangely it worked on my end with the commas but not without. Not true with github)

File: README.md (hunk 2 above)
Comment (booellean) [reply README.md]: When you say make two lines, you are referring to the section breaks/horizontal rules?

**Pull request merged**